### PR TITLE
Todo add remove

### PR DESCRIPTION
**Pull request by shella12**
###  To Do list: add & remove

I implemented the CRUD (create, read, update, delete) methods. Data is stored in localStorage.
A new JavaScript file is created and imported into the index.js as per requirement. Readme file is also editted.

### CODE REVIEW

File: src/add-remove.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+addList.addEventListener('click', () => {
+  if (inputList.value !== '') {
+    const des = inputList.value;
+    todoArray.push(
+      { description: des, completed: false },
+    );
+  }
+  inputList.value = '';
+  /* eslint-disable */
+  createList();
```
Comment (ShoiraTa) [inline src/add-remove.js]: - [ ] Please do not disable linters

File: src/add-remove.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    removeTask(list);
+  } else if (event === 'labelClick') {
+    checkboxImage.src = iconDots;
+    checkboxImage.id = 'dots';
+    list.style.backgroundColor = 'white';
+    checkboxLabel.style.backgroundColor = 'white';
+    checkboxLabel.id = 'label';
+    checkboxLabel.readOnly = true;
+    todoArray[index].description = checkboxLabel.value;
+    localStorage.setItem('data', JSON.stringify(todoArray));
```
Comment (ShoiraTa) [inline src/add-remove.js:41]: - [ ] Index property is missing in a to do task, I can see only description and completed properties. 

Screenshot 2022-08-18 at 17 59 03


Kindly add Property `index` should set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5).

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @shella12 ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- You passed all the Linters! ✔️ 
- Professional pull request! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/add-remove.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+addList.addEventListener('click', () => {
+  if (inputList.value !== '') {
+    const des = inputList.value;
+    todoArray.push(
+      { description: des, completed: false, index: 0 },
+    );
+  }
+  inputList.value = '';
+  /* eslint-disable */
+  createList();
```
Comment (gemmen29) [inline src/add-remove.js]: - [ ] Kindly try to remove the disabled linters line and fix it manually as asked by the previous reviewer.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @shella12, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 

## Hi @shella12 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ add and delete to-do functionality working as expected
✔️ No linter errors

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**